pom.xml

```diff
@@ -586,8 +591,9 @@
         <nimbus.jose.jwt.version>7.9</nimbus.jose.jwt.version>
         <org.json.version>3.0.0.wso2v1</org.json.version>
         <os.mvn.plugin.version>1.6.2</os.mvn.plugin.version>
-        <otel.alpha.version>1.6.0-alpha</otel.alpha.version>
-        <otel.version>1.6.0</otel.version>
+        <otel.alpha.version>1.18.0-alpha</otel.alpha.version>
```

Comment:
We dont need this property right?

Reply:
It is required for opentelemetry-semconv. https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-semconv